docs/implementations/luci.md

```diff
@@ -0,0 +1,16 @@
+# LUCI Implementation
+LUCI implements the WDI for USB and will be compatible with Bluetooth in the future. Due to LUCI's design and where it sits within the wheelchair electronics, there are a number of limitations with this implementation.
+
+## General
+1. LUCI turns off when the wheelchair does, so an input device cannot be used to turn the chair on through LUCI.
+1. Seating position cannot be moved using the input device through LUCI.
+1. User menu cannot be navigated using the input device through LUCI.
+1. Speed setting adjustments made by LUCI are not remembered by the wheelchair electronics. This means that the previously set speed setting will come back if the profile is changed or the the chair is rebooted, and that any changes made to speed setting through the normal joystick interface will change based on what the speed setting was before LUCI changed it (e.g. speed 5 -> LUCI changes to speed 1 -> speed setting decrease through JSM leads to speed 4).
+
+## Chairs with Tracking
+This includes most modern Permobil chairs.
+1. LUCI overwrites the post-tracking joystick. LUCI implements it's own tracking to compensate for this, but drive behavior will be different.
+1. Speed setting does not affect maximum speed.
+
+## Xbox Controllers
+1. Controllers plugged in before the chair/LUCI is turned on will **not** turn on and connect automatically. The controller has to be plugged in after LUCI already powered. This is believed to be an issue with the underlying linux driver.
```

Comment:
wording: "...after LUCI is powered. This is a known bug and believed to be an issue with the underlying Linux driver."